src/main/kotlin/no/nav/tiltaksarrangor/unleash/UnleashService.kt

```diff
@@ -1,29 +1,32 @@
 package no.nav.tiltaksarrangor.unleash
 
 import io.getunleash.Unleash
-import no.nav.amt.lib.models.deltakerliste.tiltakstype.ArenaKode
+import no.nav.amt.lib.models.deltakerliste.tiltakstype.Tiltakskode
 import org.springframework.stereotype.Component
 
 @Component
 class UnleashService(
 	private val unleash: Unleash,
 ) {
 	private val tiltakstyperKometAlltidErMasterFor = setOf(
-		ArenaKode.ARBFORB,
-		ArenaKode.INDOPPFAG,
-		ArenaKode.AVKLARAG,
-		ArenaKode.ARBRRHDAG,
-		ArenaKode.DIGIOPPARB,
-		ArenaKode.VASV,
-		ArenaKode.JOBBK,
-		ArenaKode.GRUPPEAMO,
-		ArenaKode.GRUFAGYRKE,
+		Tiltakskode.ARBEIDSFORBEREDENDE_TRENING,
+		Tiltakskode.OPPFOLGING,
+		Tiltakskode.AVKLARING,
+		Tiltakskode.ARBEIDSRETTET_REHABILITERING,
+		Tiltakskode.DIGITALT_OPPFOLGINGSTILTAK,
+		Tiltakskode.VARIG_TILRETTELAGT_ARBEID_SKJERMET,
+		Tiltakskode.JOBBKLUBB,
+		Tiltakskode.GRUPPE_ARBEIDSMARKEDSOPPLAERING,
+		Tiltakskode.GRUPPE_FAG_OG_YRKESOPPLAERING,
 	)
 
-	// her kan vi legge inn de neste tiltakstypene vi skal ta over
-	private val tiltakstyperKometKanskjeErMasterFor = emptyList<ArenaKode>()
+	private val tiltakstyperKometKanskjeErMasterFor = setOf(
+		Tiltakskode.ENKELTPLASS_FAG_OG_YRKESOPPLAERING,
+		Tiltakskode.ENKELTPLASS_ARBEIDSMARKEDSOPPLAERING,
+		Tiltakskode.HOYERE_UTDANNING,
+	)
```

Comment:
Er Komet master for disse nå?

Reply:
Nei, vi er kanskje master (se variabelnavn). Disse sammen med unleashtoggle gjør at vi kan toggle på i dev og prod som vi ønsker.